### PR TITLE
Themes: Change tests to use single runner

### PR DESCRIPTION
**Pull request by seear**
Part of #3942

### CODE REVIEW

**Comment by blowery** [comment]
woo!

Run into any crazy things?

File: client/components/themes-list/test/index.jsx [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
 
 describe( 'ThemesList', function() {
+	let React, TestUtils, ThemesList;
+	useSandbox();
 	this.timeout( 10 * 1000 );
```
Comment (blowery) [inline client/components/themes-list/test/index.jsx]: can probably drop the timeout too, we globally timeout at 10 seconds under circle now.

**Comment by blowery** [comment]
looks great! :ship: if you're happy.

File: client/components/themes-list/test/index.jsx (hunk 1 above)
Comment (gziolo) [inline client/components/themes-list/test/index.jsx]: Removed with https://github.com/Automattic/wp-calypso/pull/4163/commits/0914b5f330c9301b1644a774321fb425f9b51368 :)

**Comment by gziolo** [comment]
I added commit that removes React warnings and ESLint warnings.

**Comment by seear** [comment]
> Run into any crazy things?

Only the same issue with `useCleanCache` and React deps as @aduth encountered [here](https://github.com/Automattic/wp-calypso/pull/4088), unfortunately before I read that PR. Adding a note to the `useMockery` Readme in #4184.

**Pull request merged**